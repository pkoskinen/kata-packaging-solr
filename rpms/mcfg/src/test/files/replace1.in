line1

test %%TOBEREPLACED%% , %%TOBEREPLACED%%  done

foo
