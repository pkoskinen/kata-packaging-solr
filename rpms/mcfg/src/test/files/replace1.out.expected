line1

test replacement , replacement  done

foo
